server/sonar-server/src/test/resources/org/sonar/server/computation/activity/ActivityManagerTest/shared.xml

```diff
@@ -1,7 +1,9 @@
 <dataset>
   <projects id="10" kee="P1" qualifier="TRK" uuid="ABCD" name="Project 1"/>
   <snapshots
```

Comment:
I've made the decision to change any tag in DB unit file I change from now on to have one row per line
you did half that, why not do it all the way ?